src/methods.jl

```diff
@@ -129,11 +157,21 @@ In-place version of [`flat_correct`](@ref)
 """
 function flat_correct!(frame::AbstractArray, flat_frame::AbstractArray; norm_value = mean(flat_frame))
     norm_value <= 0 && error("norm_value must be positive")
-    frame ./= (flat_frame ./ norm_value)
+    frame ./= (flat_frame / norm_value)
```

Comment:
you shouldn't remove that second dot! Without out it you are missing out on broadcast fusion!